js_modules/dagster-ui/packages/ui-core/src/graph/SVGViewport.tsx

```diff
@@ -602,6 +602,7 @@ export class SVGViewport extends React.Component<SVGViewportProps, SVGViewportSt
           backgroundSize: `${dotsize}px`,
           cursor: isClickHeld ? 'grabbing' : 'grab',
         })}
+        className="svgViewport"
```

Comment:
I think it'd be better to use a custom data value for this, because someone is very likely to remove an unused classname. Maybe `data-svg-viewport={true}`?